Source/GoogleMapsCarbonFootprint.js

```diff
@@ -77,18 +73,17 @@ function insertFootprint(route) {
  *   - trees: amount of trees to offset the CO2 emission in one year of growth
  */
 function createElement(footprint, trees) {
-  var e = document.createElement("div");
+  var e = document.createElement('div');
   var treesStr = treesToString(trees);
-  e.innerHTML = " <a href='http://goo.gl/yxdIs' target='_blank' title='"+ 
-                treesStr + 
-                "' class='carbon' id='carbon'>" + 
+  e.innerHTML = ' <a href=\'http://goo.gl/yxdIs\' target=\'_blank\' title=\'' +
+                treesStr +
+                '\' class=\'carbon\' id=\'carbon\'>' +
                 footprintToString(footprint) +
-                "</a> <a class='offset-link' href='http://goo.gl/yxdIs' target='_blank' title='"+ 
-                treesStr + "'>offset</a>"  
+                '</a> <a class=\'offset-link\' href=\'http://goo.gl/yxdIs\' target=\'_blank\' title=\'' +
```

Comment:
Here you go: https://google.github.io/styleguide/javascriptguide.xml?showone=Strings#Strings